EXT_WAVEFRONT.md

```diff
@@ -0,0 +1,110 @@
+Domain-Specific Naming Conventions for Coherent Wavefront Propagation Codes
+===========================================================================
+
+openPMD extension name: `WAVEFRONT`
```

Comment:
If you like, you can also write this non-caps. We used CAPS for extensions that are abbreviations so far.
Others are CamelCase, e.g. `SpeciesType`.
```suggestion
openPMD extension name: `Wavefront`
```

Reply:
fine with me.